src/value.h

```diff
@@ -40,7 +40,6 @@ struct StringHeader
 		StringHeader* header = Allocate(length);
 		char* buffer = StringBuffer(header);
 		memcpy(buffer, text, length);
-		buffer[length] = 0;

```

Comment:
At first this made my Spidey Sense ™️ tingle until I saw the zero termination is done in `Allocate`.